tests/test_base_scanner.py

```diff
@@ -276,10 +276,7 @@ class SignatureTests(ScannerTestCase):
     def test_matched_signatures_are_excluded(self, mock_signature: mock.MagicMock):
         mock_signature.return_value = "foo"
         test_scanner = TestScanner(self.options)
-        self.options.exclude_signatures = ()
-        test_scanner.config_data = {
-            "exclude_findings": [{"signature": "foo", "reason": "foo finding"}]
-        }
+        self.options.exclude_signatures = ("foo",)
         self.assertTrue(test_scanner.signature_is_excluded("bar", "blah"))
```

Comment:
The new property you created is `excluded_signatures`, but here you are referencing `exclude_signatures`. That makes me think, this should not be passing.